src/libs/Browser/index.website.ts

```diff
@@ -101,4 +109,4 @@ const openRouteInDesktopApp: OpenRouteInDesktopApp = (shortLivedAuthToken = '',
     }
 };
 
-export {getBrowser, isMobile, isMobileSafari, isSafari, isMobileChrome, openRouteInDesktopApp};
+export {getBrowser, isMobile, isMobileSafari, isMobileWebKit, isSafari, isMobileChrome, openRouteInDesktopApp};
```

Comment:
Nope, we need to cover all iOS: mWeb WebKit-based browser engines, this includes iOS: Chrome, Firefox and ay other browsers which are forced by Apple to use the WebKit engine.

Therefore we need to cover for all of them, otherwise the infinite loading would appear on Chrome / Firefox / other browsers one can install on their iPhone.